crates/oxc_linter/src/rules/jsx_a11y/tab_index_no_positive.rs

```diff
@@ -63,7 +63,7 @@ fn check_and_diagnose(attr: &JSXAttributeItem, ctx: &LintContext<'_>) {
     }
 }
 
-fn parse_jsx_value(value: &JSXAttributeValue) -> Result<f64, ()> {
+pub fn parse_jsx_value(value: &JSXAttributeValue) -> Result<f64, ()> {
```

Comment:
sorry, promise this is the last one, can we move this to 

https://github.com/oxc-project/oxc/blob/6a90cd4af43ab1c083ebd81378b966a143156fda/crates/oxc_linter/src/utils/react.rs#L180

to avoid importing from another lint rule

Reply:
Fixed in https://github.com/oxc-project/oxc/pull/1795/commits/bdcd93430e513ff4cdef96b668593d4a98e888a4. Thank you for the thorough review, and I appreciate your patience with the multiple revisions. 🙏

Reply:
no worries! Thank you for contributing!